core/src/mindustry/entities/comp/BuildingComp.java

```diff
@@ -63,6 +63,7 @@
     transient boolean enabled = true;
     transient float enabledControlTime;
     transient String lastAccessed;
+    transient boolean unloadable = true;
```

Comment:
This should be an overridable `boolean` method instead of an instance variable, to be in line with how everything else is implemented.

Reply:
ok, though this was better for performance given how often its called and only for this single purpose atm, I'll get it changed 👍